src/librustc_errors/emitter.rs

```diff
@@ -1179,13 +1179,13 @@ impl EmitterWriter {
             let level_str = level.to_string();
             // The failure note level itself does not provide any useful diagnostic information
             if *level != Level::FailureNote && !level_str.is_empty() {
-                buffer.append(0, &level_str, Style::Level(level.clone()));
+                buffer.append(0, &level_str, Style::Level(*level));
```

Comment:
You might want to leave it for another PR but `level` should probably be `Level`, not `&Level`.
@petrochenkov and/or @estebank might know more about what's happening in this code though.

Reply:
I think at some point `Level` wasn't `Copy`, this is probably just a leftover from that.